angular/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+### Breaking changes
+
+*   The template parser no longer supports styles defined inside the template
+    itself.
+
+    Previously, the following two snippets would have been parsed and shimmed in
+    the same way. `html <style> .my-class {padding: 10px;} </style>`
```

Comment:
Language tags don't work on inline code like this. This will be rendered as `html <style> ...`: https://github.com/dart-lang/angular/blob/8820b34c763228fcf5ccc20ec5a5bdcfc640cec3/angular/CHANGELOG.md

Reply:
That was originally a stand-alone block. But I guess I didn't notice that the formatter merged it inline.

I'll send you a quick fix.